projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/jobData.ts

```diff
@@ -48,3 +49,19 @@ export function getJobDataJsonObject() {
   };
   return jsObj;
 }
+
+/*
+ * Function that checks whether a value is defined in jobData
+ * Shows dialog that operation cannot be performed because of undefined value
+ */
+export async function checkDefinedValue(option: VdkOption): Promise<boolean> {
```

Comment:
What is "defined value"? is this the user session data?

I opened VEP-994, and searched by "session" keyword, and didn't find anything. May you link to the description of VDK-stored data in Jupyter?

Reply:
The information about how we store data is in the README of the vdk-jupyterlab-extension. Not in the VEP. Do you think I should add more information about it in the VEP?

Reply:
Presuming a user entry-point may be the current code snippet, how do you navigate back to the VEP-994 where the VDK model in Jupyter is documented?

Reply:
I will mention it in the VEP shortly and provide a link to the READM where more information is provided

Reply:
nit: may have a bidirectional link to the VEP. However, it's strange that I cannot find it myself using the terms and wording from the code -that means there are discrepancies

Reply:
mentioned in VEP